src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -128,6 +128,11 @@ object Parameters {
   val PARAM_USE_AWS_MULTIPLE_PARTS_UPLOAD: String = knownParam(
     "use_aws_multiple_parts_upload"
   )
+  // Treat Decimal(X, 0) as Long when reading from Snowflake
+  // WARNING: if the Decimal value is greater than max value of Long, conversion will happen.
+  val PARAM_TREAT_DECIMAL_X_0_AS_LONG: String = knownParam(
```

Comment:
Can we just call this `treat_decimal_as_long`? We don't need the X.

Reply:
We can. I thought it doesn't affect Decimal with scale != 0. It looks it is not necessary.
Will update it as `TREAT_DECIMAL_X_0_AS_LONG`.